src/decode.c

```diff
@@ -464,6 +464,7 @@ void DecodeRegisterPerfCounters(DecodeThreadVars *dtv, ThreadVars *tv)
     dtv->counter_flow_udp = StatsRegisterCounter("flow.udp", tv);
     dtv->counter_flow_icmp4 = StatsRegisterCounter("flow.icmpv4", tv);
     dtv->counter_flow_icmp6 = StatsRegisterCounter("flow.icmpv6", tv);
+    dtv->counter_flow_pkts_bypassed = StatsRegisterCounter("flow.pkts_bypassed", tv);
```

Comment:
this will only work for internal bypass, right? Not for driver/hw level bypass, as then suri won't see the packets. I think this may be confusing for ppl. Perhaps the counter should depend on the actual setting we use in af-packet.